dhis-2/build-dev.sh

```diff
@@ -40,8 +40,8 @@ print() {
 print "Building dhis2-core..."
 
 export MAVEN_OPTS="-Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3 -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.test.skip=true"
-mvn clean install --batch-mode --no-transfer-progress --threads 1C -Pdev -f "$DIR"/pom.xml -pl -dhis-web-embedded-jetty
-mvn clean install --batch-mode --no-transfer-progress --threads 1C -Pdev -f "$DIR"/dhis-web/pom.xml
+mvn clean install --batch-mode --no-transfer-progress --threads 2C -DskipTests -f "$DIR"/pom.xml -pl -dhis-web-embedded-jetty
```

Comment:
`-DskipTests` only says don't run tests. Adding `-Dmaven.test.skip=true` will make sure we don't compile tests, which we do not need here 😄

Reply:
Reverting the remove of dev profile.
We are doing too many things in this PR and the dev profile is also used in the dhis-web projects.
In dev profile tests are already skipped, I will maintain it that way